charts/platform-tenant/values.yaml

```diff
@@ -69,18 +70,65 @@ kyverno:
 # If loki, prometheus and grafana installed
 # this configures a namespaced dashboard.
 o11y:
-  enable: true
+  enabled: true
   # namespace where prometheus resides
   namespace: o11y
   labels: {}
   annotations: {}
 
+# appAuth configures all external (and internal) access to for named applications inside
+# of the tenant namespace.
+appAuth:
```

Comment:
Let's structure this like the `platform-system` chart where it's under `components.oauth2Proxy`. We'll have more `components` v soon.